cnv_release_notes/cnv_release_notes.adoc

```diff
@@ -11,6 +11,17 @@ include::modules/cnv_introduction_to_cnv.adoc[leveloffset=+2]
 [[RN_new_features]]
 == New and changed features
 
+==== Updated KubeVirt API
+** A new version of the KubeVirt API is included in {ProductName} 1.4. Several 
+important changes are reflected in the latest configuration file templates.
+*** The `apiVersion` has been updated from `kubevirt.io/v1alpha2` to 
+`kubevirt.io/v1alpha3`.
+*** The `volumeName` attribute no longer exists. Ensure that each disk name 
+matches the corresponding volume name in all configuration files.
+*** All instances of `registryDisk` have been updated to read `containerDisk`.
```

Comment:
+ "Ensure that all instances of registryDisk get update to containerDisk"

My pont, a user needs to update this in their definitions.

Reply:
@fabiand Upgrading from 1.x->1.4 is not supported, correct? Would a user still have files with "registryDisk" to manually update?